core/print_api.php

```diff
@@ -1678,7 +1702,11 @@ function print_email_link_with_subject( $p_email, $p_text, $p_tooltip, $p_bug_id
 	$t_mailto = string_attribute( 'mailto:' . $t_email . '?subject=' . $t_subject );
 	$t_text = string_display( $p_text );
 
-	echo '<a href="' . $t_mailto . '"' . $t_tooltip . '>' . $t_text . '</a>';
+        if( $p_icon_only == true ) {
+                echo '<a href="' . $t_mailto . '"' . $t_tooltip . ' class="fa fa-envelope-o" style="text-decoration: none;"></a>';
```

Comment:
What's the purpose of the inline style ? We try to avoid these.

Reply:
Used to disable the underscore fa fa-envelope-o when you hover the pointer.
Unfortunately, I am still very weak in the structure of css mantisbt.
I would be very grateful if you tell me how to implement it correctly.